gazebo_ros/launch/gzserver.launch.py

```diff
@@ -53,7 +53,13 @@ def generate_launch_description():
         _plugin_command('force_system'), ' ',
         # Wait for (https://github.com/ros-simulation/gazebo_ros_pkgs/pull/941)
         # _plugin_command('force_system'), ' ',
-        _arg_command('profile'), ' ', LaunchConfiguration('profile'),
+        _arg_command('profile'), ' ', LaunchConfiguration('profile'), ' ', 
+        # convenience parameter for params file
+        PythonExpression([
+            '"--ros-args', ' --params-file', '" if "" != "',
+            LaunchConfiguration("params_file"), '" else ""' ]),
+        LaunchConfiguration("params_file"), ' ',
+
```

Comment:
I think we better terminate `--ros-args` with `--`, otherwise if we launch with `extra_gazebo_args` set, it may be confused for ROS arguments.

Also, it looks like if we drop the extra spaces this works without setting `shell=True` on `ExecuteProcess`.

```suggestion
        '--ros-args',
        PythonExpression([
            '"--params-file', '" if "" != "',
            LaunchConfiguration("params_file"), '" else ""' ]),
        LaunchConfiguration("params_file"),
        '--',
```